### ISSUE TITLE
`mup setup` fails silently if there are relative paths in `mup.js`

### ISSUE THREAD

**Issue by maxko87** [zodern/meteor-up#339]
This issue had me stumped for a while. It would be nice to raise an error if there are relative paths in the mup file.

**Comment by Deadlyweed** [comment]
@maxko87 Can you post your mup.js for me please.

**Comment by geeksville** [comment]
I just happened upon this as well.  Here's the relevant non working snippet in my mup.js:

```
meteor: {
    name: 'persistence',
    path: '../persistence'
}
```

Here's a snippet that works: (my mup.js is in /home/kevinh/development/persistence/.deploy, pwd is .deploy when I ran mup setup)

```
meteor: {
    name: 'persistence',
    path: '/home/kevinh/development/persistence'
}
```

Hopefully this note helps.

**Comment by Deadlyweed** [comment]
@geeksville Do you work on it local or a development server? Will you be running more then one app from the server?

**Comment by geeksville** [comment]
on a development server.  I already run two other meteor apps on that server (which I'm currently working on transitioning to the new docker style mup - I previously used the old 0.x mup)

I don't yet have my new docker based solution working though - I can update this bug after I finish (hopefully later today - now fighting with https://github.com/meteor/meteor/issues/7475 )

**Comment by Deadlyweed** [comment]
@geeksville Thank you, I added that to my readme file. Trying to make one that is way easier to understand for everyone

**Comment by geeksville** [comment]
Thanks so much.  Everything is working nicely for me now.  Just so you know remaining things I had to do to get things happy: (I can file other bugs or send PRs if you prefer)
- The fix listed above
- I'm using meteor 1.4.x so I needed to use abernix/meteord:base per #7475 as my starting container, without that it would fail to start because the node inside the default container was too old. (Later I had to customize the container anyways to pull in various native dependencies for my app)

Thanks for the awesome tool!
